### PR TITLE
Update bangladesh medical import script to ensure safe imports

### PR DESCRIPTION
**Pull request by govindkrjoshi**
The PR fixes the following issues while importing data for bangladesh:

- Fix patient gender to `male`, `female` or `transgender`
- Ensure either `date_of_birth` or (`age` and `age_updated_at`) are present
- Add patient status
- Add patient phone number status
- Assign `unknown` to nil medical history questions
- Add metadata and metadata_version to patient business identifiers
- Add an empty rxnorm_code to prescription drugs

### CODE REVIEW

**Comment by harimohanraj89** [comment]
We need to add `receiving_treatment_for_hypertension` to [this list](https://github.com/simpledotorg/simple-server/blob/master/lib/bangladesh/to_patients.rb#L137)

File: lib/bangladesh/to_patients.rb [hunk 1]
```diff
@@ -56,35 +60,69 @@
...
 end
 
+def patient_recorded_at(blood_pressures)
+  date = blood_pressures.map { |bp| bp[:recorded_at] }.min
+  return DateTime.new(2019, 6, 1) if date.nil?
+  date
+end
+
 def create_patient(params)
+  if params[:gender].blank?
```
Comment (kitallis) [inline lib/bangladesh/to_patients.rb]: Any idea why this doesn't work? Does the sheet have spaced strings?

Comment (timcheadle) [reply lib/bangladesh/to_patients.rb]: We moved the gender check to where the CSV is parsed to fix this

**Comment by harimohanraj89** [comment]
Closing in favor of #809

**Pull request closed without merging**